cmd/up.go

```diff
@@ -401,19 +408,82 @@ func (cmd *UpCmd) devPodUpMachine(
 			client.AgentPath(),
 			client.AgentURL(),
 			true,
-			command,
+			sshCmd,
 			stdinReader,
 			stdoutWriter,
 			writer,
 			log.ErrorStreamOnly(),
 		)
-		if err != nil {
+		if err != nil && !errors.Is(err, context.Canceled) && !strings.Contains(err.Error(), "signal: ") {
 			errChan <- fmt.Errorf("executing agent command: %w", err)
 		} else {
 			errChan <- nil
 		}
 	}()
 
+	// create pipes
+	stdoutReader2, stdoutWriter2, err := os.Pipe()
```

Comment:
could you rename these and `stdinReader/Writer2` below pls? It's currently not clear what they are being used for without scrolling down

Reply:
Done.